### DISCUSSION TITLE
Discussions area for Collaborators only?

### DISCUSSION THREAD

**Discussion by Plasma** [community/community#18427] [Discussions]
Hi team,

I recently joined as a Collaborator for a project, and was thinking it would be nice to speak directly to the other collaborators to orient myself, prior to drafting a PR, but was thinking it would be good to have that discussion just with other collaborators.

Just wanted to leave this as a feature suggestion, so a discussion area can be marked visible to Collaborators only.

**Comment by ascarobiondo007** [comment]
Inviato da Posta <https://go.microsoft.com/fwlink/?LinkId=550986> per
Windows



*Da: *Andrew Armstrong ***@***.***>
*Inviato: *domenica 12 giugno 2022 14:18
*A: *github-community/community ***@***.***>
*Cc: *Subscribed ***@***.***>
*Oggetto: *[GitHub Community] Discussions area for Collaborators only?
(Discussion #18427)



Hi team,

I recently joined as a Collaborator for a project, and was thinking it
would be nice to speak directly to the other collaborators to orient
myself, prior to drafting a PR, but was thinking it would be good to have
that discussion just with other collaborators.

Just wanted to leave this as a feature suggestion, so a discussion area can
be marked visible to Collaborators only.

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/github-community/discussions/18427>, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ANEBHXDLQ3NLASOKTWXKY33VOXIR7ANCNFSM5YRU4SGA>
.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>
